skimage/segmentation/manual_segmentation.py

```diff
@@ -0,0 +1,187 @@
+import numpy as np
+import matplotlib
+import matplotlib.pyplot as plt
+from matplotlib.patches import Polygon
+from matplotlib.collections import PatchCollection
+from ..draw import polygon
+
+
+def _mask_from_verts(verts, shape):
+    mask = np.zeros(shape, dtype=bool)
+    pr = [y for x, y in verts]
+    pc = [x for x, y in verts]
+    rr, cc = polygon(pr, pc, shape)
+    mask[rr, cc] = 1
+    return mask
+
+
+def manual(image, alpha=0.4, return_masks=False, draw_lines=False):
+    """Return a binary image based on the selections made with mouse clicks.
+
+    Parameters
+    ----------
+    image : (M, N[, 3]) array
+        Grayscale or RGB image.
+
+    alpha : float, optional
+        Transparancy value for polygons draw over the segments.
```

Comment:
Would be nice to change color for the new contour/polygon. Maybe iterate over `plt.cm.Spectral` or something?